.github/workflows/pysa.yml

```diff
@@ -0,0 +1,34 @@
+name: pysa
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  pysa:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8.7
+
+      - name: Install pyre-check
+        run: |
+          VERSION=$(grep "version" .pyre_configuration | sed -n -e 's/.*\(0\.0\.[0-9]*\).*/\1/p')
+          pip install pyre-check-nightly==$VERSION
+
+      - name: Run & Test Pysa
+        run: |
+          cd ./documentation/deliberately_vulnerable_flask_app
+          . ./setup.sh
+          printf '.\n' | pyre init
+          # Add excludes to .pyre_configuration, modifies taint-models-path to ../../stubs
+          echo '{"source_directories": ["."],"exclude": [".*\/site-packages\/.*"]}' > ./.pyre_configuration
```

Comment:
I don't think so, I removed it primarily because it was taking a hell lot of time, say like 17 minutes to run and reported additional vulnerabilities that were not part of the `delibrately_vulnerable_flask_app` . And removing it makes the build quicker and report all 3 issues that we know to be true.

Reply:
I was wrong here! we needed it only when a virtual environment is "in" `delibrately_vulnerable_flask_app` folder. When it is so, pysa goes on checkingfunctions in the virtual env and gets in a sort of loop. Moving outside the virtual environment in github actions (or moving the virtual environment outside the folder) makes us not want this line. I've fixed it now!